Dockerfile

```diff
@@ -1,12 +1,25 @@
 # renovate: datasource=docker depName=renovate/renovate
 ARG RENOVATE_VERSION=24.116.2
 
-FROM renovate/renovate:${RENOVATE_VERSION}-slim
+FROM renovate/renovate:${RENOVATE_VERSION}-slim as distsource
```

Comment:
@rarkins Should we remove the slim dependency and diret build here? so we get the full images faster?

Reply:
@viceice I decided to go with this approach to make sure that renovate source and dependencies are for sure the same across slim and full image.

Reply:
I think we can rely on the immutable nature of npm and build this one directly

Reply:
+ docker-entrypoint.sh is currently present in slim repo, but not here, so using slim image helps avoiding copy-paste it here..

Reply:
@viceice @rarkins please let me know if I need to switch to installing package instead of copying it from `slim` version. As I mentioned this would also require holding separate version of `docker-entrypoint.sh` in repo

Reply:
@rarkins some transitive dependencies can diff between slim and full image. I think this is acceptable

@zeldigas Yes, please copy all required files here and build from npm.

Reply:
@rarkins @viceice I tried to do suggested change, but amount of copy-paste looks pretty huge. Could you please take a look at this commit: https://github.com/zeldigas/docker-renovate-full/commit/1e5139752d6b6dc3e7f6578a6d3aae8f7969c7c3. If you still think that this approach is better, I'll add this commit to PR.

Reply:
Yes, please add that, but remove commented code from dockerfile.

Reply:
Done